### PR TITLE
Update about_us

### PR DESCRIPTION
**Pull request by kalebphipps**
Two small changes:
- Added Stefan.
- Added the link to the paper.

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos)
- [X] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Did you update the CHANGELOG

## PR review
Anyone in the community is free to review the PR once the tests have passed.

- [X] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [X] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones to the PR so it can be classified

### CODE REVIEW

**Comment by kalebphipps** [comment]
Funding should now be ok (I hope)

File: docs/source/about_us.rst [hunk 1]
```diff
@@ -39,4 +39,4 @@ Benedikt Heidrich, Andreas Bartschat, Marian Turowski, Oliver Neumann, Kaleb Phi
 
 Funding
 -------
-This project is supported by the Helmholtz Association under the Program “Energy System Design” and Helmholtz Association’s Initiative and Networking Fund through Helmholtz AI as well as by the German Research Foundation (DFG) Research Training Group 2153 "Energy Status Data: Informatics Methods for its Collection, Analysis and Exploitation".
+This project is supported by the Helmholtz Association under the Program “Energy System Design”, by Helmholtz Association’s Initiative and Networking Fund through Helmholtz AI, by the Helmholtz Association under the Joint Initiative "Energy System 2050 - A Contribution of the Research Field Energy", and by the German Research Foundation (DFG) Research Training Group 2153 "Energy Status Data: Informatics Methods for its Collection, Analysis and Exploitation".
```
Comment (marianturowski) [inline docs/source/about_us.rst:42]: ..., by the Helmholtz...

**Review by marianturowski — CHANGES_REQUESTED** [review]
Please add a "the" in line 42

**Comment by kalebphipps** [comment]
"the"  added :)

**Review by marianturowski — APPROVED** [review]

**Pull request merged**